tests/unit/__init__.py

```diff
@@ -0,0 +1 @@
+test contents
```

Comment:
These line with plain "tests content" are not syntactically correct. You can use module level docstring.